easybuild/toolchains/linalg/intelmkl.py

```diff
@@ -80,6 +80,23 @@ def __init__(self, *args, **kwargs):
         class_constants.extend(['BLAS_LIB_MAP', 'SCALAPACK_LIB', 'SCALAPACK_LIB_MT', 'SCALAPACK_LIB_MAP'])
         super(IntelMKL, self).__init__(*args, **kwargs)
 
+    def set_variables(self):
+        """Set the variables"""
+
+        # for recent versions of Intel MKL, -lm -ldl should be used for linking
+        # according to the Intel MKL Link Advisor, this should always be done,
+        # but it only seems to be strictly required for certain compilers
+        mkl_version = self.get_software_version(self.BLAS_MODULE_NAME)[0]
+        if LooseVersion(mkl_version) >= LooseVersion('11'):
+        #if LooseVersion(mkl_version) >= LooseVersion('11') and self.COMPILER_FAMILY in [TC_CONSTANT_PGI]:
```

Comment:
The math library and the dynamic linking library? If these are not around, we will have bigger concerns that this. Sure, `glibc-devel` is needed but that's a requirement anyway.